src/DesktopTests/Settings/SnapshotMetaInfoTests.cs

```diff
@@ -100,16 +98,12 @@ public void Deserialize_SecondPixel_Null()
 
         private static Dictionary<SnapshotMetaPropertyName, object> GetDeserializedProperties(Dictionary<SnapshotMetaPropertyName, object> inputProperties)
         {
-            using (ShimsContext.Create())
+            SnapshotMetaInfo info = new SnapshotMetaInfo(A11yFileMode.Contrast,"Test", 0, 0, inputProperties);
+            var text = JsonConvert.SerializeObject(info);
```

Comment:
>text [](start = 16, length = 4)

The Shim was only being used to return this from the StreamReader. By using it to initialize the Stream, we get the same result without a Shim